src/plugins/kibana/public/settings/sections/indices/add_data_steps/pipeline_setup/lib/processor_types.js

```diff
@@ -0,0 +1,12 @@
+export default [
+  { // set
+    typeId: 'set',
+    title: 'Set',
+    targetField: '',
+    value: '',
+    getDescription: function (processor) {
```

Comment:
If this was a class and getDescription was an instance method, it could be implemented as an actual getter method so it could be accessed as `processor.description`.